presidio-analyzer/tests/test_certificate_number_recognizer.py

```diff
@@ -18,10 +18,10 @@ def entities():
     "text, expected_len, expected_scores, expected_res",
     [
         # fmt: off
-        ("AS1111111 D91111111", 2, (), ((0, 9),(10,19),),),
-        ("AD2222222", 1, (), ((0, 9),),),
-        ("my certificate number is: AD2222222", 1, (), ((26,35),),),
-        ("The DEA number is  A61111111", 0, (), (),),
+        ("GL0285191 EU4488929", 2, (), ((0, 9),(10,19),),),
```

Comment:
Can we add those cases in addition and not instead? Is there a different reason for replacing?

Reply:
the checksum calculation was broken, so the test was not actually valid licenses, I replaced them with valid ones according to https://www.meditec.com/blog/dea-numbers-what-do-they-mean

Reply:
Thanks. Looks good